community/neo4j/src/docs/ops/upgrades.asciidoc

```diff
@@ -4,89 +4,77 @@
+The location of these files is specified by configuring the `org.neo4j.server.database.location` variable in `conf/neo4j-server.properties`.
+====
 
-. Install Neo4j {neo4j-version}, and set it up to use the same database store directory (typically _data/graph.db_).
+[[supported-upgrade-paths]]
+== Supported upgrade paths
 
-. Make a copy of the database.
-+
-[IMPORTANT]
-It is strongly advised to make a copy of the database store directory at this time, to use as a backup in case rollback/downgrade is required.
-This is not necessary if a backup has been made using the
-ifndef::upgradetext[<<operations-backup, online backup tool>>, ]
-ifdef::upgradetext[online backup tool (see http://neo4j.com/docs/{neo4j-version}/operations-backup.html), ]
-available with Neo4j Enterprise.
+Before upgrading to a new major or minor release, the Neo4j database must first be upgraded to the latest patch release within the relevant major and minor release.
+You can find out the relevant patch release number for your installation at this page: {neo4j-releases-download-page}.
+The following Neo4j upgrade paths are supported, where _x_ stands for the latest patch release within a specific minor release, and _y_ stands for any patch release number:
 
-. Start up Neo4j.
+* 1.9.x -> {neo4j-version}
 
-. Any database store upgrade required will occur during startup.
+* 2.0.x -> {neo4j-version}
 
-[[explicit-upgrade]]
-== Explicit Store Upgrade
+* 2.1.x -> {neo4j-version}
 
-To perform an explicit store upgrade:
+* 2.2.x -> {neo4j-version}
 
-. Install Neo4j {neo4j-version}, and set it up to use the same database store directory (typically _data/graph.db_).
-. Cleanly shut down the older version of Neo4j, if it is running.
-. Set the Neo4j configuration parameter `allow_store_upgrade=true` in your _conf/neo4j.properties_ file.
-  Neo4j will fail to start without this configuration set.
-. Start up Neo4j.
-. The database store upgrade will occur during startup.
-. The `allow_store_upgrade` configuration parameter should be removed, set to `false` or commented out.
-. Information about the upgrade and a progress indicator are logged into the _messages.log_ file, inside the database store directory.
+* 2.3.y -> {neo4j-version}
 
 [CAUTION]
 .Disk space requirements
 ====
-An explicit upgrade will require substantial free disk space, as it must make an entire copy of the database store.
-The upgraded store version may also require larger store files overall.
-It is suggested to have available free disk space equivalent to at least 1.5 times the size of the existing store.
+An upgrade of major or minor releases requires substantial free disk space, as it makes an entire copy of the database.
+The upgraded database may also require larger data files overall.
+It is recommended to have available free disk space equivalent to at least 1.5 times the size of the existing database.
 ====
```

Comment:
The mention of "major and minor releases" here is the only distinction we make between those and point releases anywhere on the page. In the interests of simplicity I suggest we don't draw that distinction here.